test/compiler/_relational.out

```diff
@@ -0,0 +1,10 @@
+False
+True
+True
+True
+True
+True
+True
+True
+False
+False
```

Comment:
see `_relational.py` - this is just the expected output of running the compiled script.
the input file `_relational.ods` is compiled through our workflow, the python is checked against `_relational.py`, and then the output is checked against `_relational.out`.